server/src/main/scala/io/delta/standalone/internal/DeltaSharedTableLoader.scala

```diff
@@ -643,6 +730,45 @@ class DeltaSharedTable(
       new Path(path, p)
     }
   }
+
+  private def computeChecksum(queryParamChecksum: QueryParamChecksum): String = {
+    DigestUtils.sha256Hex(JsonUtils.toJson(queryParamChecksum))
+  }
+
+  private def decodeAndValidatePageToken(
+                                          tokenStr: String,
+                                          expectedChecksum: String): QueryTablePageToken = {
+    val token = decodeQueryTablePageToken(tokenStr)
+    val timestampFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss z")
+    timestampFormat.setTimeZone(TimeZone.getTimeZone("UTC"))
```

Comment:
nit: will the UTC time confuse the user who sees the error message?

Reply:
Removed.